null_safety/calculate_lix/bin/main.dart

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) 2020, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+import 'package:calculate_lix/lix.dart' as lix;
```

Comment:
I'd perform the 'sort directives' operation here; it'll add a newline between the dart: and package: import.